content/collections/experiment-results/en/experiment-results-use-formula-metrics.md

```diff
@@ -123,6 +123,18 @@ This function only works when grouping by a numerical property on the last event
 
 Returns the average value of the given property among all the users who completed the conversion. This function internally uses Funnel Query to get the average value among the converted users.
 
+### PROPCOUNT
+
+**Syntax:** `PROPCOUNT(event)`
+
+* **Event:** Refers to the event that interests you. This must be a letter that corresponds to an event in the Events card. If grouping by multiple properties, the formula runs the calculation with the first group by clause.
+
+Returns the number of distinct property values for the property the event is grouped by. In this setup, the formula retrieves the number of different departments covering all the items for which details were viewed:
+
+![propcount_sidecontrols.png](/docs/output/img/event-segmentation/propcount-sidecontrols-png.png)
+
+`PROPCOUNT` is an **estimate** of distinct property values. This estimate comes from a [HyperLogLog algorithm,](https://en.wikipedia.org/wiki/HyperLogLog) and its accuracy depends on amount of data it has to work with. Expect a relative error in the range of 0.1% for less than 12,000 unique values, and up to 0.5% for more than 12,000 unique property values, depending on the cardinality of the property. 
```

Comment:
can you also add that we only support numeric event properties in experiment but analytics supports any data type as the event property you are counting